tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist = py{35,36,37,38}-sphinx{20,21,22,23}
+
+[testenv]
+passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
+deps=
+    -rdev-requirements.txt
```

Comment:
This is an internal implementation detail and is subject to change.
The canonical way (as per the docs) to install the development requirements is `pip install -e .[dev]`.